urdfpy/utils.py

```diff
@@ -180,6 +186,19 @@ def unparse_origin(matrix):
     return node
 
 
+def resolve_filepath(base_path, file_path):
+    parsed_url = urlparse(file_path)
+    dirname = base_path
+    file_path = parsed_url.netloc + parsed_url.path
+    while not dirname == '/':
+        resolved_filepath = os.path.join(dirname, file_path)
+        print(resolved_filepath)
+        if os.path.exists(resolved_filepath):
+            return resolved_filepath
+        dirname = os.path.dirname(dirname)
+    return False
```

Comment:
in the tests, this will return false, then in `get_filename`, `os.path.isabs` fails because it's trying to evaluate `False`